### PR TITLE
added edit course test

### PR DESCRIPTION
**Pull request by jmaldonado-rojas**

### CODE REVIEW

File: e2e/pages/editCourse.po.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        });
+        return this;
+    }
+
+    clickSaveButton(){
+        CommonActions.click('input#edit-submit');
+        return this;
+    }
+
+    getConfirmationMessage(){
```
Comment (CristelhM) [inline e2e/pages/editCourse.po.js]: better to move this to commons, seems like 'CommonActions.getText('div.message-text');' is been used in different pages

File: e2e/specs/editCourse.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        let updatedCourse = {
+            'CourseName': 'Other Name',
+            'SectionName': 'Other Section',
+            'SubjectArea': 'Technology',
+            'Level' : 'None'
+        };
+
+        browser.url(env.url);
+        let header = login.loginAs(env.credentials.teacher.username, env.credentials.teacher.password);
```
Comment (CristelhM) [inline e2e/specs/editCourse.spec.js]: you may move these two lines to @before:
 browser.url(env.url);
let header = login.loginAs(env.credentials.teacher.username, env.credentials.teacher.password);

File: e2e/specs/editCourse.spec.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        let editCourse = header.navigateTo('Courses')
+            .clickCreateCourseButton()
+            .fillCreateCourseForm(course)
+            .clickCreateButton()
+            .clickCourseOptions()
+            .clickEditInfoLink()
+            .fillEditCourseInfoForm(updatedCourse)
+            .clickSaveButton();
+
+        expect(editCourse.getConfirmationMessage()).to.equal('The section has been updated.');
```
Comment (CristelhM) [inline e2e/specs/editCourse.spec.js]: you may need to add validations to each modified field, not just the confirmation message

**Review by CristelhM — CHANGES_REQUESTED** [review]

File: e2e/specs/createCourse.spec.js [hunk 4]
```diff
@@ -12,8 +19,6 @@ describe('Create Course', () => {
             'Level' : 'None'
         };
 
-        browser.url(env.url);
-        let header = login.loginAs(env.credentials.teacher.username, env.credentials.teacher.password);
         let courseDashboard = header.navigateTo('Courses')
```
Comment (CristelhM) [inline e2e/specs/createCourse.spec.js]: you can use constants instead of hardcoded values

File: e2e/specs/editCourse.spec.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        'Level' : 'None'
+    };
+
+    before(() => {
+        header = login.loginAs(env.credentials.teacher.username, env.credentials.teacher.password);
+
+    });
+
+    beforeEach(() => {
+        courseDashboard = header.navigateTo('Courses')
```
Comment (CristelhM) [inline e2e/specs/editCourse.spec.js]: you can use constants instead of hardcoded values

**Review by CristelhM — CHANGES_REQUESTED** [review]

File: e2e/pages/editCourse.po.js (hunk 1 above)
Comment (jmaldonado-rojas) [reply e2e/pages/editCourse.po.js]: moved!!

File: e2e/specs/editCourse.spec.js (hunk 2 above)
Comment (jmaldonado-rojas) [reply e2e/specs/editCourse.spec.js]: moved!!

File: e2e/specs/editCourse.spec.js (hunk 3 above)
Comment (jmaldonado-rojas) [reply e2e/specs/editCourse.spec.js]: added

File: e2e/specs/createCourse.spec.js (hunk 4 above)
Comment (jmaldonado-rojas) [reply e2e/specs/createCourse.spec.js]: updated

File: e2e/specs/editCourse.spec.js (hunk 5 above)
Comment (jmaldonado-rojas) [reply e2e/specs/editCourse.spec.js]: updated

File: e2e/specs/editAndRemoveCourse.spec.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    let course = {
+        'CourseName': 'Test 201',
+        'SectionName': 'Section 1',
+        'SubjectArea': 'Technology',
+        'Level' : 'None'
+    };
+
+    before(() => {
+        header = login.loginAs(env.credentials.teacher.username, env.credentials.teacher.password);
+
```
Comment (CristelhM) [inline e2e/specs/editAndRemoveCourse.spec.js:19]: blank kine

File: e2e/specs/editAndRemoveCourse.spec.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    beforeEach(() => {
+        courseDashboard = header.navigateTo(feature.COURSES)
+            .clickCreateCourseButton()
+            .fillCreateCourseForm(course)
+            .clickCreateButton();
+    });
+
+    it('it should be possible to edit a course', () => {
+
```
Comment (CristelhM) [inline e2e/specs/editAndRemoveCourse.spec.js:30]: blank line

File: e2e/specs/editAndRemoveCourse.spec.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+const login = require('../pages/login.po');
+const env = require('../../environment');
+const expect = require('chai').expect;
+const feature = require('../core/constants').feature;
+
+describe('Edit and remove a course', () => {
+
```
Comment (CristelhM) [inline e2e/specs/editAndRemoveCourse.spec.js:7]: blank

**Review by CristelhM — CHANGES_REQUESTED** [review]

**Pull request merged**